nuxeo-drive-core/src/test/java/org/nuxeo/drive/service/adapter/TestFileSystemItemAdapterService.java

```diff
@@ -638,14 +647,6 @@ public void testContribOverride() throws Exception {
         assertTrue(activeFactories.contains("dummyFacetFactory"));
         assertTrue(activeFactories.contains("dummyVirtualFolderItemFactory"));
         assertTrue(activeFactories.contains("nullMergeTestFactory"));
-
-        Framework.getRuntime().standby(Instant.now());
-        try {
-            harness.undeployContrib("org.nuxeo.drive.core.test",
```

Comment:
Don't you need to undeploy the contribution ? see https://github.com/nuxeo/nuxeo-drive-server/pull/35#discussion_r119891244